src/main/kotlin/com/ampnet/headlesscmsservice/persistence/model/Text.kt

```diff
@@ -0,0 +1,28 @@
+package com.ampnet.headlesscmsservice.persistence.model
+
+import java.util.UUID
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.Id
+import javax.persistence.Table
+
+@Entity
+@Table(name = "text")
+class Text(
```

Comment:
It would be nice to rename this to `Content` because `Text` is a limiting name and it is also used inside a model. 
Also, didn't we agree on using API route: `/content` instead of `/text` or it's just my thought? @mightymatth

Reply:
Me and @mightymatth agreed on text afterwards, but sure `Content` might be more suitable, I agree.